uptane-standard.md

```diff
@@ -818,7 +818,7 @@ The ECU SHALL create a version report as described in {{version_report}}, and se
 
 ### Metadata verification procedures {#metadata_verification}
 
-A Primary ECU MUST perform full verification of metadata. A Secondary ECU SHOULD perform full verification of metadata. If a Secondary cannot perform full verification, it SHALL perform partial verification instead.
+A Primary ECU MUST perform full verification of metadata. A Secondary ECU SHOULD perform full verification of metadata. If a Secondary cannot perform full verification, it SHALL perform at least partial verification instead.
```

Comment:
Same reservation about "at least": there's either partial or full, I don't understand what a subset would look like.

Reply:
See explanation above. We are trying to establish that there is a "minimum" that must be performed. If that point is not clear, perhaps it needs to be rephrased.

Reply:
It's the word order in this case. See suggestion:
```suggestion
A Primary ECU MUST perform full verification of metadata. A Secondary ECU SHOULD perform full verification of metadata. If a Secondary cannot perform full verification, it SHALL, at the very least, perform partial verification.
```